addons/account_edi_ubl_cii/models/account_edi_xml_ubl_20.py

```diff
@@ -572,12 +572,9 @@ def _export_invoice(self, invoice):
         xml_content = self.env['ir.qweb']._render(vals['main_template'], vals)
         return etree.tostring(cleanup_xml_node(xml_content), xml_declaration=True, encoding='UTF-8'), set(errors)
 
-    def _get_document_type_code_vals(self, invoice, invoice_data):
-        """Returns the values used for the `DocumentTypeCode` node"""
-        # To be overriden by custom format if required
-        # http://www.datypic.com/sc/ubl20/e-cbc_DocumentTypeCode.html
-        document_type_code = 380 if invoice.move_type == 'out_invoice' else 381
-        return {'attrs': {}, 'value': document_type_code}
+    def _get_document_type_code_node(self, invoice, invoice_data):
```

Comment:
This change will break the stable policy. 
If a partner module is using this piece of code (by overriding it or something), their code will break.
Please, keep the first function (`_get_document_type_code_vals`) and add a check if we have to create the `DocumentTypeCode` node or not.